src/to_wiki.js

```diff
@@ -118,23 +168,48 @@ const serializer = new WikiTextSerializer({
    },
 
    text(state, node) {
-      let marks = node.marks || []
+      let marks = node.marks.map(m => m.type.name)
```

Comment:
[pedantic] I think this would be easier to read if this variable was named `markNames`.